frontend/src/Constants/InitialFarmDetails.ts

```diff
@@ -0,0 +1,12 @@
+// Initial Values for calculation, some defaults are being used
+import FarmDetailsInterface from 'src/Interface/FarmDetailsInterface';
+
+const initialFarmDetails: FarmDetailsInterface = {
+  Year: '',
+  FarmName: '',
+  FarmRegion: '',
+  HasBerries: true,
+  Fields: [{ FieldName: '', Area: 0, Comments: null }],
+};
+
+export default initialFarmDetails;
```

Comment:
Good job on putting this on a different file. Maintaining this would be hard on a lot of lines.

Reply:
It also improves reusability :wink: